src/SDKs/Authorization/Authorization.Tests/Authorization.Tests.csproj

```diff
@@ -20,6 +20,9 @@
   </ItemGroup>
 
   <ItemGroup>
+    <None Update="SessionRecords\Authorization.Tests.BasicTests\ProviderOperationsMetadataListGetTests.json">
```

Comment:
@Monicalyy please replace all these lines with something like below.
This helps in maintaining project files concise.
< ItemGroup>
    < None Update="SessionRecords \ ** \ *.json">
      < CopyToOutputDirectory>PreserveNewest< /CopyToOutputDirectory>
    < /None>
< /ItemGroup>